### PR TITLE
Authentication routes for signup and login

### PR DESCRIPTION
**Pull request by XinranTaoAngela**
currently since there's no database implemented, the backend enables user to use any password and username to login.

### CODE REVIEW

**Comment by byroncjy** [comment]
Looks good, just see the one comment about logout route

**Review by byroncjy — COMMENTED** [review]
Looks good, just see the one comment about logout route

**Comment by larahynkim** [comment]
Hi Angela! I think everything looks good, please just make sure that both .env files in the back-end and the front-end are to port 3001. I think the back-end env file was to 3001 and the front-end env file was to 3000 in the files you posted on discord, so there was a communication error between the be and the fe. thanks for your work!

**Review by larahynkim — APPROVED** [review]
Hi Angela! I think everything looks good, please just make sure that both .env files in the back-end and the front-end are to port 3001. I think the back-end env file was to 3001 and the front-end env file was to 3000 in the files you posted on discord, so there was a communication error between the be and the fe. thanks for your work!

File: back-end/authRoutes.js
[the remark this answers is not in the document]
Comment (XinranTaoAngela) [reply back-end/authRoutes.js:1]: Good point

**Review by byroncjy — APPROVED** [review]
Approved - logout route checked

**Pull request merged**